src/client.js

```diff
@@ -72,6 +73,10 @@ export class UiClient {
         this._showWidget(mfaCredentialsWidget, options);
     }
 
+    showMfaList(options) {
```

Comment:
```suggestion
    showMfaCredendials(options) {
```

Reply:
Don't commit this through Github, there's a typo.

Reply:
🕵️